File: counting_characters.exs
defmodule CountingCharacters do
  def run do
    input = IO.gets("What is your input string? ") |> String.trim()

    num_chars = String.length(input)

    response = "#{input} has #{num_chars} characters."

    IO.puts(response)
  end
end

CountingCharacters.run()

File: counting_characters.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)

	fmt.Print("What is your input string? ")
	input, _ := reader.ReadString('\n')

	input = strings.TrimSpace(input)

	numChars := len(input)

	response := fmt.Sprintf("%s has %d characters.", input, numChars)

	fmt.Println(response)
}
